### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by KatyaZaremska**
[DEMO LINK](https://KatyaZaremska.github.io/react_person/)

### CODE REVIEW

**Comment by KatyaZaremska** [comment]
[DEMO LINK](https://katyazaremska.github.io/react_person/)

**Comment by KatyaZaremska** [comment]
[DEMO LINK](https://katyazaremska.github.io/react_person/)

File: src/App.jsx [hunk 1]
```diff
@@ -25,21 +25,8 @@ export const alex = {
...
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person person={misha} />
```
Comment (33Orange) [inline src/App.jsx:28]: You didn't import Person to app.jsx

image
image

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,25 @@
...
+      {age
+        && (
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+        )
+      }
+      <p className="Person__partner">
+        {isMarried ? (
+          `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
```
Comment (33Orange) [inline src/components/Person/Person.jsx:20]: Create variable for this `sex === 'm' ? 'wife' : 'husband'`

**Review by 33Orange — COMMENTED** [review]
Almost done

A few moments to fix

**Pull request closed without merging**